src/76usedatabase.js

```diff
@@ -10,203 +10,149 @@
 /* global yy alasql */
 
 // CREATE DATABASE databaseid
-yy.CreateDatabase = function (params) {
-	return Object.assign(this, params);
-};
+yy.CreateDatabase = (params) => Object.assign(this, params); // Updated to arrow function
```

Comment:
Please keep regular functions for high level so they remain informative in the stack trace